### ISSUE TITLE
"firstpass" regex causes labels not to be removed (@rollup/plugin-strip)

### ISSUE THREAD

**Issue by trbrc** [rollup/plugins#167]
- Rollup Plugin Name: @rollup/plugin-strip
- Rollup Plugin Version: 1.3.1
- Rollup Version: 1.28.0
- Node Version: 12.13.1

### How Do We Reproduce?

REPL.it reproduction: https://repl.it/repls/KookyMicroGui

### Expected Behavior

All labels `y` should be removed, as specified in the `strip` config in `rollup.config.js`.

### Actual Behavior

The `y` label in `input/fails.js` is not removed.

---

The bug is this check, which causes an early return when processing `fails.js`: https://github.com/rollup/plugins/blob/d3db3bbdd0981bb5c5a830f23c7b7dd730a17a66/packages/strip/src/index.js#L55

Which happens because the `firstpass` regex only looks for the function names, not labels:
https://github.com/rollup/plugins/blob/d3db3bbdd0981bb5c5a830f23c7b7dd730a17a66/packages/strip/src/index.js#L47

In this case, the config is falling back on the default `{functions: ['console.*', 'assert.*']}`. Since `fail.js` doesn't contain `console` or `assert` or `debugger`, the plugin incorrectly skips processing the labels.

**Comment by shellscape** [comment]
Thanks for reporting this and for the great reproduction and info. We'll get on it as soon as possible. If you have an immediate need and would like to PR a fix, we'd be happy to review one!

**Comment by trbrc** [comment]
I did a PR with what I think is the right fix, but would be grateful if someone who is already familiar with the test setup would add a test.

In the mean time I found that it's easy to work around. Simply use the plugin twice, once for the labels, once for the functions, and make sure to specify `functions: []` to override the default for the instance that strips labels.

(As an aside, if I may: I think the `['console.*', 'assert.*']` default is a pretty bad idea, at least if the config specifies `labels`. It's very easy to overlook. It certainly made this particular issue a lot more difficult to troubleshoot, as it made it appear as if labels were being stripped or kept at random.)
